README.md

```diff
@@ -58,6 +58,14 @@ If you want to set host to a specific IP address for example `172.17.12.1` run `
 
 *To view visit http://localhost:8000/demos/<name>*
 
+## Guidelines for CSS Development
+
+- For issues created in Core that will affect a component in PF-React, a follow up issue must be created in PF-React once the Pull Request is merged.
```

Comment:
Do we want to outline what information would be useful for the issue that is being created? PR #, Core Release, link to core component as well as any information detailing the change?

Reply:
Yes I think that a good idea :)